packages/cms/tsconfig.json

```diff
@@ -5,5 +5,5 @@
     "outDir": "./dist/client"
   },
   "include": ["**/*.ts"],
-  "exclude": ["./dist/", "./src/studio/**"]
+  "exclude": ["./dist/"]
```

Comment:
I think `studio` is external code that should stay excluded.

Reply:
Scratch this. Didn't realize this wasn't pointing to the actual sanity studio output. Shouldn't have been excluded to begin with.